packages/react-components/components/PhoneNumberInput.tsx

```diff
@@ -69,40 +69,56 @@ export default class PhoneNumberInput extends React.Component<Props, State> {
     }
   }
 
+  async getPhoneNumberFromNativePicker() {
+    // linter doesn't allow following code without splitting this code in two lines
```

Comment:
Why? What's the error?

Reply:
Does `return SmsRetriever.requestPhoneNumber()` work? There will be an implicit await in the return because it's an async function. That should fix it if it was the `no-return-await` rule

Reply:
oh, coo, didn't know the `await` was implicit!